projects/plugins/jetpack/extensions/blocks/premium-content/_inc/subscription-service/class-token-subscription-service.php

```diff
@@ -48,15 +48,9 @@ public function initialize() {
 	 * How to obtain one of these (or what exactly it is) is
 	 * still a WIP (see api/auth branch)
 	 *
-	 * @inheritDoc
-	 *
-	 * @param array $valid_plan_ids List of valid plan IDs.
-	 * @param array $access_level Access level for content.
-	 *
-	 * @return bool Whether the user can view the content
+	 * @return array|false The token payload or false otherwise
 	 */
-	public function visitor_can_view_content( $valid_plan_ids, $access_level ) {
-
+	public function get_token_payload() {
```

Comment:
Nice!

Reply:
Maybe we could also change it here

Reply:
nice, let's note it for a follow up PR.

Reply:
Maybe it could be integrated in this one? It would also add another user of `get_token_payload()` and legitimate its creation even more ;)